apiserver/storage/storage.go

```diff
@@ -256,12 +256,17 @@ func filterAttachment(machineSet set.Strings, attachment volume.Attachment) (par
 	}
 
 	size := attachment.Size()
+
+	storageTag := names.StorageTagKind + "-unknown/0"
+	if names.IsValidStorage(attachment.Storage()) {
```

Comment:
Thank you for your explanation - I was not aware that it is valid for volumes not to have a storage id when unassigned.
Changed to panic for invalid storage id but not to panic or try to get a tag when storage id is not supplied :D